app/services/preserver/blind_importer/file_metadata_adapter/query_service.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+class Preserver::BlindImporter::FileMetadataAdapter
+  class QueryService
```

Comment:
A typical valkyrie query service will reference an external converter but b/c of the limited scope here it's all rolled into this class. not just in the form of ConvertLocalStorageIDs but also via `Valkyrie::Types::Anything`